portfolio/src/main/webapp/index.html

```diff
@@ -1,86 +1,118 @@
+                            My name is Sumanjit Gill and I am a third-year Computer Engineering student attending CSU
+                            Fresno.
+                            <br><br>
+                            I enjoy any activity involving creativity, be it drawing, photography, video editing,
+                            animation, or programming.
+                        </p>
+
                         <br>
-                        I enjoy any activity involving creativity, be it drawing, photography, video editing, animation, or programming.
-                    </p>
+
+                    </div>
                 </div>
-            </div>
 
-            <p>Click below for a quote from <i>Avatar: The Last Airbender!</i></p>
-            <button class = "button" onclick="addRandomATLAQuote()">Quote Me!</button>
-            <div id="quote-container"></div>
+                <p>Click below for a quote from <i>Avatar: The Last Airbender!</i></p>
+                <button class="button" onclick="addRandomATLAQuote()">Quote Me!</button>
+                <div id="quote-container"></div>
 
-            <br>
+                <br>
 
-            <div class = "row">
-                <b><h4>Contact Me:</h4></b>
-                    <div class = "column-main">
+                <div class="row">
+                    <b>
+                        <h4>Contact Me:</h4>
+                    </b>
+                    <div class="column-main">
                         <form action="/contact" method="POST">
-                            <label for = "name">Name</label>
+                            <label for="name">Name</label>
                             <br>
-                            <input type = "text" id="name" name = "name" placeholder="Your Name" required>
+                            <input type="text" id="name" name="name" placeholder="Your Name" required>
                             <br><br>
 
-                            <label for = "email">Email</label>
+                            <label for="email">Email</label>
                             <br>
-                            <input type = "text" id="email" name = "email" placeholder="Your Email" required>
+                            <input type="text" id="email" name="email" placeholder="Your Email" required>
                             <br><br>
 
-                            <label for = "message">Message</label>
+                            <label for="message">Message</label>
                             <br>
-                            <input type = "text" id="message" name = "message" placeholder="Your Message" required>
+                            <input type="text" id="message" name="message" placeholder="Your Message" required>
                             <br><br>
-                            <br/>
-                            <input type="submit"/>
+                            <br />
+                            <input type="submit" />
                         </form>
+
```

Comment:
Instead of using the form, how about adding a separate text box only used by the translate function? I think this would help the API function feel like it's own stand-alone feature and provide a better UX. 

Either way, I'd recommend checking out <textarea> for the translate message box. This gives the user a larger (and expandable) canvas to write on. Note the Translate API has a 100K bytes limit on the request you send. Nothing to worry about in this case - but more of an FYI.